hw/ip_templates/racl_ctrl/rtl/racl_ctrl.sv.tpl

```diff
@@ -243,6 +243,12 @@ module ${module_instance_name} import ${module_instance_name}_reg_pkg::*; #(
 
   `ASSERT_KNOWN(RaclErrorKnown_A, racl_policies_o)
 
-  // Alert assertions for reg_we onehot check
-  `ASSERT_PRIM_REG_WE_ONEHOT_ERROR_TRIGGER_ALERT(RegWeOnehotCheck_A, u_reg, alert_tx_o[0])
+  `ASSERT_PRIM_REG_WE_ONEHOT_ERROR_TRIGGER_ALERT_IN(RegWeOnehotCheck_A,
+                                                    u_reg,
+% if enable_shadow_reg:
+                                                    gen_alert_tx[1].u_prim_alert_sender.alert_req_i)
```

Comment:
I think it's always index 0. But better use `AlertFatalFaultIdx` to index

Reply:
You're completely right. It turns out that this commit had been on my work branch for a while. I was about to say "I think it probably predates 40cf3b0bab8" (16th July), then looked at the author date on the commit. I wrote it at the end of May!

(Tut, tut, Rupert: don't forget to push PRs!)

I've just made the obvious fix and have also rebased onto the head of master.